tests/cxx/comment_style-d1.h

```diff
@@ -0,0 +1,22 @@
+//===--- comment_style-d1.h - test input file for iwyu --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// this file exists to verify show_comments works
+#ifndef COMMENT_STYLE_D1_H
```

Comment:
Include guards in IWYU tests are usually: `INCLUDE_WHAT_YOU_USE_TESTS_CXX_COMMENT_STYLE_H_` (see other test cases). But there's also a tendency to avoid them entirely if they're not needed, and I don't think they are here.